akka-actor/src/main/scala/akka/util/ConstantFun.scala

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) 2015-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.util
+
+import akka.annotation.InternalApi
+import akka.japi.function.{ Function ⇒ JFun, Function2 ⇒ JFun2 }
+import akka.japi.{ Pair ⇒ JPair }
+
+/**
+ * INTERNAL API
+ */
+@InternalApi private[akka] object ConstantFun {
```

Comment:
Vote for copy and pasting.
For the critical bugfix, that maybe only for commercial users:)